Week2/homework/js-exercises/evenOdd.js

```diff
@@ -0,0 +1,4 @@
+'use strict'
+for(let i=0; i<=20; i++){
+    (i%2===0) ? console.log("The number " + i + " is even!") : console.log("The number " + i + " is odd!")
+}
```

Comment:
I would like to recommend using a normal `if-else` loop here instead of conditional operators. In general, we should use conditional operators only in situations where we need to extract a value from the condition and do an assignment operation. For example, using a conditional operator makes sense in situations like: `const isItCold = weather < 20 ? 'Yes' : 'No'`. Makes sense?